WcaOnRails/app/models/competition.rb

```diff
@@ -1136,7 +1136,8 @@ def self.search(query, params: {}, managed_by_user: nil)
       competitions = competitions.where(like_query, part: "%#{part}%")
     end
 
-    competitions.includes(:delegates, :organizers).order(start_date: :desc)
+    date_order = params[:date_order] == "asc" ? :asc : :desc
```

Comment:
We actually used to support a `sort` parameter to this API, which we removed in https://github.com/thewca/worldcubeassociation.org/pull/803/. You can see a bit of a discussion about this over in https://github.com/thewca/worldcubeassociation.org/pull/803#discussion_r73791543.

I think I still prefer the `sort=start_date`/`sort=-start_date` to sort by asc/desc start date. One nice thing about it is it would allow us to more easily add support for sorting by multiple fields in the future (like if you want to break start_date ties on end_date, the user could specify `sort=start_date,end_date`).

Reply:
Good point, done!